src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +46,9 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(element::equals)
+                .findAny().orElseThrow(NoSuchElementException::new);
```

Comment:
```suggestion
                .findAny()
                .orElseThrow(NoSuchElementException::new);
```
take a look to orElseThrow() without parameters

Reply:
done